### ISSUE TITLE
Trained using the Semantic KITTI dataset, but did not achieve the official accuracy.

### ISSUE THREAD

**Issue by jeannotes** [xieKKKi/MotionBEV#5]
hi, I trained using the Semantic KITTI dataset, but did not achieve the official accuracy. (bs: 20, lr: 0.025) (bs: 8, lr: 0.025) all not achieve the official accuracy
but with your livox dataset, use your parameter, I get the accuracy, can you give me advice

**Comment by xieKKKi** [comment]
> hi, I trained using the Semantic KITTI dataset, but did not achieve the official accuracy. (bs: 20, lr: 0.025) (bs: 8, lr: 0.025) all not achieve the official accuracy but with your livox dataset, use your parameter, I get the accuracy, can you give me advice

https://github.com/xieKKKi/MotionBEV/blob/233a3484a103bfd0f71e693c9db82da8a6e4a94b/config/MotionBEV-semantickitti.yaml#L49 The default learning rate set in MotionBEV-semantickitti.yaml is 0.01.
btw, can you get the expected results using the pre-trained weights?
